dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -27,6 +27,8 @@
 using NavigationViewItemSeparator = Microsoft.UI.Xaml.Controls.NavigationViewItemSeparator;
 using NavigationViewBackButtonVisible = Microsoft.UI.Xaml.Controls.NavigationViewBackButtonVisible;
 using System.Collections.ObjectModel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
```

Comment:
Looks like this line caused build failures. We already have it at L18 so I think you can just remove it.

Reply:
Good catch, removed it now.